content/articles/line-detection-using-hough-transform-in-matlab/index.md

```diff
@@ -0,0 +1,149 @@
+### Line detection using hough transform in Matlab
+### Table of content
+[Introduction](#introduction)
+[Prerequisites](#prerequisites)
+[Standard hough transform](#standard-hough-transform)
+[Using hough transform to detect a line](#using-hough-transform-to-detect-a-line)
+[Conclusion](#conclusion)
+
+### Introduction
+Hough transform is an algorithm that isolates specific shapes in an image. These desired features should are specified in a given parametric form. This algorithm is widely used in detecting regular curves. These curves are such as lines, curves and ellipses etc. However, the general application of hough transform is impossible to attain the simple analytical feature description. For example, line detection is an important feature that helps analyze two objects. It can give a more important relationship between the two objects.
+This tutorial will look at how we can use the hough transform in Matlab. We will understand the importance of line detection and how to carry this out using Matlab code.
+
+### Prerequisites
+To follow along with this tutorial, you will need:
+- [MATLAB](https://www.mathworks.com/products/get-matlab.html?s_tid=gn_getml) installed.
+- Proper understanding of [MATLAB](https://www.section.io/engineering-education/getting-started-with-matlab/) basics.
+
+### Standard hough transform
+We can use this transform to measure the length of a line. For example, suppose a line in an image needs to be detected; it can do that.
+
```

Comment:
```suggestion
### Standard hough transform
We can use this transform to measure the length of a line. For example, suppose a line in an image needs to be detected, it can do that.

```